src/ServiceBouncer/MainForm.cs

```diff
@@ -321,6 +324,19 @@ private void SetTitle()
             }
         }
 
+        private void SetConnectedStatusBar()
+        {
+            if (isActive)
+            {
+                var backgroundRefreshTimeText = backgroundRefreshSeconds == 1 ? "1 second" : $"{backgroundRefreshSeconds} seconds";
+                toolStripStatusLabel.Text = $@"Connected to {machineHostname}. - Background refresh every {backgroundRefreshTimeText}.";
+            }
+            else
+            {
+                toolStripStatusLabel.Text = $@"Connected to {machineHostname}.";
```

Comment:
Could this message state no refresh please.

Reply:
Remove the refresh status message completely?

Reply:
Sorry, no, that wasn't a clear comments.
Can you set the text to "Connected..... Background Refresh Disabled"

Reply:
Yes, of course :)